src/main/java/com/devcycle/sdk/server/local/api/DVCLocalClient.java

```diff
@@ -113,40 +110,43 @@ public <T> Variable<T> variable(User user, String key, T defaultValue) {
             .isDefaulted(true)
             .build();
 
-    if (!isInitialized) {
+    if (!configManager.isConfigInitialized() || !isInitialized) {
```

Comment:
Do we need to handle this in the Java side, what does the WASM return if you call it b4 things are setup?

Reply:
Indeed, i think there is probably another way of doing it but I wasn't 100% the right approach given the different libs used in the project. I was going to ask Kaushal about it.

I think it needs to be done this way for two reasons. The JSON deserialization is unable to generate a generically typed Variable<T> object directly, so we need to deserialize and then wrap it. Secondly, the Variable objects in this SDK are read only (just private methods) so you can't set the default flag and value after the fact, requiring a new object to be created.

Reply:
This is a pattern we are already doing in DotNet and Go but I noticed that the java client didn't handle this situation. The WASM throws an exception when you try to call variable() and no configuration was set. 

```
Unable to evaluate Variable string-var due to error: io.github.kawamuray.wasmtime.WasmtimeException: error while executing at wasm backtrace:
    0: 0xe758 - <unknown>!<wasm function 172>
    1: 0x19e9b - <unknown>!<wasm function 307>
    2: 0x1ae50 - <unknown>!<wasm function 317>
    3: 0x2884f - <unknown>!<wasm function 448>
```

Reply:
I've noticed that this isInitialized field on DVCClients in a few of the SDKs isn't really useful.  It just gets set to true if the constructor successful completes but doesn't represent anything else. It would be nice if it actually represented that the client object is fully initialized (including dependencies) and ready to use